src/main/java/org/folio/circulation/domain/UpdateRequestQueue.java

```diff
@@ -11,15 +12,16 @@
 import java.util.concurrent.CompletableFuture;
 import java.util.stream.Collectors;
 
-import org.folio.circulation.resources.context.ReorderRequestContext;
-import org.folio.circulation.support.Clients;
-import org.folio.circulation.support.ClockManager;
-import org.folio.circulation.support.Result;
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import org.folio.circulation.resources.context.ReorderRequestContext;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.ClockManager;
+import org.folio.circulation.support.Result;
```

Comment:
The [provisionally agreed import ordering standard](https://issues.folio.org/browse/FOLIO-1667) does not separate the `org.folio` packages from other `org.*` packages.

Why have they been separated in this pull request?

@mreno-EBSCO Please correct me if I am misremembering the provisional standard

Reply:
That is my understanding as well.

I hope we can get checkstyle put in place at some point to prevent these editor-based import reorders. Until then, I prefer we just leave the existing import order in a file alone as it just clutters up the commit and a different editor will likely want to change it back or to something else.

Reply:
Yeah, there likely needs to be a project wide reordering of the imports in conjunction with static analysis to identify this.

I don’t want to do that until we can get agreement on the standard.